src/chi-vue/src/components/column-customization/ColumnCustomization.tsx

```diff
@@ -96,20 +97,47 @@ export default class ColumnCustomization extends Vue {
     if (this._ColumnCustomizationContentComponent) {
       this._availableColumns = [];
       this._selectedColumns = [];
+      this._selectedData = this.columnsData?.columns.filter((column: DataTableColumn) => column.selected);
       this._processData();
-      (this.$refs.saveButton as HTMLButtonElement).disabled = true;
+      (this.$refs.saveButton as HTMLButtonElement).disabled = false;
       (this.$refs.resetButton as HTMLButtonElement).disabled = true;
       this.key += 1;
     }
-    this.$emit(DATA_TABLE_EVENTS.COLUMNS_RESET, [...(this._selectedColumns || [])]);
```

Comment:
Why removing the event emission? How will the applications using this component detect changes in column selection / order ?

Reply:
What we want is to let the user reset and continue changing columns, no reset and save instant, so we detect the change just saving.

Reset only returns the previous state.